### ISSUE TITLE
AttributeError: 'StdLoggerProxy' object has no attribute 'trace' ...when using containerlog patch() with Uvicorn/FastAPI

### ISSUE THREAD

**Issue by pwheimann** [vapor-ware/containerlog#25]
I'm attempting to use containerlog w/ FastAPI/Uvicorn, when I use path() to override the Uvicorn logs it seems to work, but on requests to my FastAPI application I get the following error a couple times on each request in the logs...

```
Exception in callback H11Protocol.connection_made(<_SelectorSoc...e, bufsize=0>>)
handle: <Handle H11Protocol.connection_made(<_SelectorSoc...e, bufsize=0>>)>
Traceback (most recent call last):
  File "/Users/paulhazadhub/.pyenv/versions/3.8.6/lib/python3.8/asyncio/events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
  File "/Users/paulhazadhub/Projects/api/gc-api/.venv/lib/python3.8/site-packages/uvicorn/protocols/http/h11_impl.py", line 129, in connection_made
    self.logger.log(TRACE_LOG_LEVEL, "%sConnection made", prefix)
  File "/Users/paulhazadhub/Projects/api/gc-api/.venv/lib/python3.8/site-packages/containerlog/proxy/std.py", line 260, in log
    getattr(self, name.lower())(msg, *args, **kwargs)
AttributeError: 'StdLoggerProxy' object has no attribute 'trace'
```

Python 3.8.6 (I saw the same behavior using 3.9.1)
FastAPI 0.62.0
Uvicorn 0.13.1 
H11 0.11.0

Running the app via `uvicorn gc_api.main:api --reload`

Just trying a simple FastAPI with patching for the Uivcorn logging as a test...running on Mac OSX Catalina 10.15.7.
```
from fastapi import FastAPI
from containerlog.proxy.std import patch

from .routes import geocode
from .routes import reverse_geocode

patch()

api = FastAPI()
api.include_router(geocode.router)
api.include_router(reverse_geocode.router)
```

It appears to not be an issue really (the error message), but does mess the logs up...and its not logged at all when you don't patch the Uvicorn logs w/ containerlog.

**Comment by edaniszewski** [comment]
Thanks for the report and all the context around it (:

I think I have an idea of what the error may be. I'll look into it a bit later today and  hopefully have a fix in for it / new release either later tonight or tomorrow morning

**Comment by edaniszewski** [comment]
This may take me a bit longer. It looks like a trace level is being dynamically added to the standard logger levels (either by h11 or uvicorn), but is not there by default, so I'll need to figure out how to add in support for dynamically set log levels. may take a few days.

**Comment by pwheimann** [comment]
Thanks...I'm going with just not patching the Uvicorn loggers for now...just putting my app logs via containerlogs...see how that turns out in ELK. I appreciate you looking into so quickly. Nice work on this lib though...very easy for python logger!

**Comment by edaniszewski** [comment]
I've got a fix for this merged into master, will cut a new release with it now

**Comment by edaniszewski** [comment]
The updates should be available in the `0.3.1` release
